data.json

```diff
@@ -398,12 +398,6 @@
       },
       "navbar-sidebar-only": {
         "navbar": {
-          "logo": {
```

Comment:
This fixes the "hidden graphic" on the sidebar examples on the components page.

The example was rendering an empty logo as the example did not have one. I ensured that this did not break other sidebars across the site

See #388